src/App.tsx

```diff
@@ -1,39 +1,135 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import Carousel from './components/Carousel';
 
-interface State {
-  images: string[];
-}
-
-class App extends React.Component<{}, State> {
-  state = {
-    images: [
-      './img/1.png',
-      './img/2.png',
-      './img/3.png',
-      './img/4.png',
-      './img/5.png',
-      './img/6.png',
-      './img/7.png',
-      './img/8.png',
-      './img/9.png',
-      './img/10.png',
-    ],
-  };
-
-  render() {
-    const { images } = this.state;
-
-    return (
-      <div className="App">
-        {/* eslint-disable-next-line */}
-        <h1>Carousel with {images.length} images</h1>
-
-        <Carousel />
+const images = [
+  './img/1.png',
+  './img/2.png',
+  './img/3.png',
+  './img/4.png',
+  './img/5.png',
+  './img/6.png',
+  './img/7.png',
+  './img/8.png',
+  './img/9.png',
+  './img/10.png',
+];
+
+const DEFAULT_VALUES = {
+  itemWidth: 130,
+  frameSize: 3,
+  step: 3,
+  animDur: 1000,
+};
+
+const App: React.FC = () => {
+  const [itemWidth, setItemWidth] = useState(DEFAULT_VALUES.itemWidth);
+  const [frameSize, setFrameSize] = useState(DEFAULT_VALUES.frameSize);
+  const [step, setStep] = useState(DEFAULT_VALUES.step);
+  const [animDur, setAnimDur] = useState(DEFAULT_VALUES.animDur);
+  const [infinite, setInfinite] = useState(false);
+  const [currentImage, setCurrentImage] = useState(0);
+
+  return (
+    <div className="App">
+      <h1 data-cy="title" className="App__title">
+        Carousel with
+        {images.length}
+        {' '}
+        images
```

Comment:
You can simplify this lines to such one:

```suggestion
       {`Carousel with ${images.length} ${images}`}
```